clients/http/reading_test.go

```diff
@@ -141,7 +141,7 @@ func TestQueryReadingsByDeviceNameAndResourceNamesAndTimeRange(t *testing.T) {
 	defer ts.Close()
 
 	client := NewReadingClient(ts.URL)
-	res, err := client.ReadingsByDeviceNameAndResourceNamesAndTimeRange(context.Background(), deviceName, resourceNames, start, end, 1, 10)
+	res, err := client.ReadingsByDeviceNameAndResourceNamesAndTimeRange(context.Background(), deviceName, resourceNames, start, end, start, end)
```

Comment:
Why was this change as part of this PR?

Reply:
Right, it's a redundant change, will revert.